### PR TITLE
{vis}[GCCcore/12.3.0] GTK3 v3.24.37, Wayland v1.22.0

### PR DESCRIPTION
**Pull request by stefan-wolfsheimer**
depends on:
* https://github.com/easybuilders/easybuild-easyconfigs/pull/18574
* https://github.com/easybuilders/easybuild-easyconfigs/pull/18584
* https://github.com/easybuilders/easybuild-easyconfigs/pull/18495
* https://github.com/easybuilders/easybuild-easyconfigs/pull/17937

### CODE REVIEW

**Comment by stefan-wolfsheimer** [comment]
The package builds on our system without the patch
`adwaita-icon-theme-3.34.3_disable-svg-conversion.patch`

I have adopted the patch to adwaita-icon-theme v44.0. 
Does it build on other systems? Does it fail due to a SVG encoding error?  Should I add the adopted patch to the PR?

**Comment by boegel** [comment]
@stefan-wolfsheimer #13449 gives more context on why the patch was added, does that help to clarify the need for it?

**Comment by stefan-wolfsheimer** [comment]
> @stefan-wolfsheimer #13449 gives more context on why the patch was added, does that help to clarify the need for it?

The original patch disabled SVG support. It compiled on our system without the patch. I don't know exactly why it was there in the first place.
The patch has been adopted to the new version, and I can add it at any time if needed.

`diff -aur a/configure.ac b/configure.ac
As we don't have SVG support we disable the search for the converter.
Adopted from:
Author: Alexander Grund (TU Dresden)

--- a/configure.ac      2023-08-18 17:58:28.032333478 +0200
+++ b/configure.ac      2023-08-18 17:59:51.554807635 +0200
@@ -49,7 +49,7 @@
 AM_CONDITIONAL(ALLOW_RENDERING, test "x$allow_rendering" = "xyes")

 symbolic_encode_sizes="16x16 32x32"
-AC_PATH_PROG([GTK_ENCODE_SYMBOLIC_SVG], [gtk-encode-symbolic-svg], [false])
+GTK_ENCODE_SYMBOLIC_SVG="false"
 if test "x$GTK_ENCODE_SYMBOLIC_SVG" = "xfalse"; then
    symbolic_encode_sizes=""
 fi`

**Comment by stefan-wolfsheimer** [comment]
> @stefan-wolfsheimer #13449 gives more context on why the patch was added, does that help to clarify the need for it?

@boegel, thanks for the hint. It's a bit clearer. I added the patch to the PR. 
Next time, I will follow the trace to PRs to get more information.

**Review by boegel — DISMISSED** [review]
lgtm

**Comment by boegel** [comment]
@boegelbot please test @ generoso
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node3118.skitty.os - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/boegel/4bca6c6fb75673b2e48400a7a7761b14 for a full test report.

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**FAILED**
Build succeeded for 1 out of 2 (2 easyconfigs in total)
zen2-ubuntu-2304 - Linux Ubuntu 23.04 (Lunar Lobster), x86_64, AMD EPYC 7452 32-Core Processor (zen2), Python 3.11.4
See https://gist.github.com/SebastianAchilles/3bd126935de36654b3a98e420db1fc3d for a full test report.

File: easybuild/easyconfigs/g/GTK3/GTK3-3.24.37-GCCcore-12.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+ provides all the user interface controls, or widgets, used in a common
+ graphical application. Its object-oriented API allows you to construct
+ user interfaces without dealing with the low-level details of drawing and
+ device interaction.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '12.3.0'}
+
+builddependencies = [
+    ('binutils', '2.40'),
```
Comment (SebastianAchilles) [inline easybuild/easyconfigs/g/GTK3/GTK3-3.24.37-GCCcore-12.3.0.eb:17]: Without Autotools installed in the OS the build failes with: `/bin/bash: line 1: autoreconf: command not found`. I would recommend to add Autotools to the build dependencies.

```suggestion
    ('binutils', '2.40'),
    ('Autotools', '20220317'),
```

**Review by SebastianAchilles — CHANGES_REQUESTED** [review]

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen2

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
zen2-ubuntu-2304 - Linux Ubuntu 23.04 (Lunar Lobster), x86_64, AMD EPYC 7452 32-Core Processor (zen2), Python 3.11.4
See https://gist.github.com/SebastianAchilles/cd73eff6f0dce7ed1be83ee0bd1b92c1 for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen2

**Review by boegel — DISMISSED** [review]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node3103.skitty.os - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/boegel/961654de2bb1cfa25827fd9425080fcc for a full test report.

**Comment by boegel** [comment]
Going in, thanks @stefan-wolfsheimer!

**Pull request merged**